### ISSUE TITLE
Mac not supported?

### ISSUE THREAD

**Issue by Sanchay-T** [rohittp0/GramUp#45]
<img width="1359" alt="Screenshot 2023-04-14 at 9 44 45 PM" src="https://user-images.githubusercontent.com/64085789/232099335-a38213e9-efd4-41b9-9983-8dfb36b1794a.png">
This is the issue

**Comment by rohittp0** [comment]
Are you on M1/M2 ?

**Comment by Sanchay-T** [comment]
M2

**Comment by rohittp0** [comment]
Yeah that should be the issue, I haven't tested on M series chips. I am rewriting the whole code and adding in a GUI that should also fix this problem.

**Comment by rohittp0** [comment]
The new release should be ready in a couple of days
